archlinuxcn/sendanywhere/PKGBUILD

```diff
@@ -0,0 +1,53 @@
+# Maintainer: Yizheng Hu <huyizheng@hotmail.com>
+# Contributor: Ye Shu <shuye02@outlook.com>
+# Contributor: Aaron Fischer <mail@aaron-fischer.net>
+# Contributor: Ashwin Vishnu <ashuwish+arch at gmail dot com>
+
+pkgname=sendanywhere
+pkgver=20.11.6
```

Comment:
final step: it seemed we need a `pkgver()` function to automatically update the version from that extracted deb file...